lib/peeps.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'database_connection'
+
+class Peeps
+
+  attr_reader :id, :peep, :date
+  
+  def initialize(id:, peep:, date:)
+    @id = id
+    @peep = peep
+    @date = date
+  end
+
+  def self.create(peep:)
+    result = DatabaseConnection.query(
+      "INSERT INTO peeps (message) VALUES ('#{peep}') RETURNING id, message, date;"
```

Comment:
Check this step walkthrough from the challenge to protect this agains SQL injections - https://github.com/makersacademy/course/blob/main/apprenticeships_bookmark_manager/walkthroughs/12.md